src/main/java/core/basesyntax/models/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.models;
+
+import core.basesyntax.Figure;
+
+public class IsoscelesTrapezoid implements Figure {
+    private final String color;
+    private final double height;
+    private final double firstBase;
+    private final double secondBase;
+
+    public IsoscelesTrapezoid(String color, double height, double firstBase, double secondBase) {
+        this.color = color;
+        this.height = height;
+        this.firstBase = firstBase;
+        this.secondBase = secondBase;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Color: " + color

```

Comment:
There are missing spaces in the concatenated string. This will cause the output to be a single continuous line without proper spacing between the words. Consider adding spaces at the end of each line before the concatenation operator.